002-es-modules.md

```diff
@@ -0,0 +1,540 @@
+| Title  | ESM Interoperability |
+|--------|-----------------------------|
+| Author | @bmeck                      |
+| Status | DRAFT                       |
+| Date   | 2016-01-07                  |
```

Comment:
maybe, unclear on this. Is it the date when things started or for last update / errata